afdko/PKGBUILD

```
@@ -3,58 +3,63 @@
 
 pkgname=afdko
 pkgver=4.0.1
-pkgrel=1
-pkgdesc='Adobe Font Development Kit for OpenType'
+pkgrel=6
+pkgdesc="Adobe Font Development Kit for OpenType"
 arch=(x86_64)
 url="https://github.com/adobe-type-tools/$pkgname"
-license=(custom)
+license=('Apache-2.0')
```

Nice fix. Lets just use the quote style in the rest of the PKGBUILD. It's a fixed string with no variable replacements and no spaces:

```suggestion
license=(Apache-2.0)
```